NameLayerMain/src/vg/civcraft/mc/namelayer/listeners/PlayerListener.java

```diff
@@ -13,23 +13,37 @@
 import org.bukkit.event.Listener;
 import org.bukkit.event.player.PlayerJoinEvent;
 
+import vg.civcraft.mc.namelayer.GroupManager.PlayerType;
 import vg.civcraft.mc.namelayer.NameLayerPlugin;
 import vg.civcraft.mc.namelayer.database.GroupManagerDao;
 import vg.civcraft.mc.namelayer.group.Group;
+import vg.civcraft.mc.namelayer.permission.GroupPermission;
+import vg.civcraft.mc.namelayer.permission.PermissionType;
 
 public class PlayerListener implements Listener{
 
 	private static Map<UUID, List<Group>> notifications = new HashMap<UUID, List<Group>>();
 	
-	@EventHandler(priority=EventPriority.LOWEST)
+	@EventHandler(priority=EventPriority.HIGHEST)
```

Comment:
Why the priority change? LOWEST means it happens first, HIGHEST means it happens second-to-last...

probably makes no difference, but can't see any reason to change it, either.